### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by matsduf**
Updated as preparation for release:
* Debian 7 removed
* Debian 8 added
* Debian 9 added
* Ubuntu 18.04 added
* FreeBSD 10.4 replaces 10.3
* Perl version on FreeBSD updated
* Perl versions and database engines have to be determined
* GUI details have to be determined

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,17 +166,7 @@ repository, i.e. [general issues in Zonemaster](https://github.com/dotse/zonemas
...
-  [dotse/zonemaster-cli#64](https://github.com/dotse/zonemaster-cli/issues/64).
-* Selecting language in Zonemaster Backend RPC API under FreeBSD does not work, see issue
-  [dotse/zonemaster-backend#315](https://github.com/dotse/zonemaster-backend/issues/315).
-* [Zonemaster-GUI version v1.0.10](https://github.com/dotse/zonemaster-gui/releases/tag/v1.0.10) fixes a
-  vulnerability in non-Javascript GUI. This is part of Zonemaster version
-  [v2017.4.1](https://github.com/dotse/zonemaster/releases/tag/v2017.4.1).
-* [Zonemaster-GUI version v1.0.11](https://github.com/dotse/zonemaster-gui/releases/tag/v1.0.11) fixes a
-  vulnerability in GUI. This is part of Zonemaster version
-  [v2017.4.2](https://github.com/dotse/zonemaster/releases/tag/v2017.4.2).
+None.
```
Comment (mattias-p) [inline README.md]: I think this section should be removed entirely unless there is a process for keeping it up to date. The process should include some guideline for what issues are to be included.

Comment (matsduf) [reply README.md]: There is, see https://github.com/dotse/zonemaster/blob/master/docs/internal-documentation/maintenance/ReleaseProcess.md#9-update-zonemaster-repository-main-readmemd

**Comment by matsduf** [comment]
@sandoche2k, can you review?

File: README.md (hunk 1 above)
Comment (mattias-p) [reply README.md]: It says that the section should be updated if needed, but there's nothing on how to make that determination.

**Comment by mattias-p** [comment]
@matsduf I'm not sure if you've noticed, but you've got some conflicts to resolve.

File: README.md [hunk 2]
```diff
@@ -44,28 +44,26 @@ architecture listed below.
...
+* Ubuntu 18.04
 
 ### Supported database engine versions
 
 Operating System | MySQL | PostgreSQL
 ---------------- | ------| -----------
 CentOS 7         | 5.6   |   9.3
-Debian 7         | 5.5   |  10.1
+Debian 8         | TBD   |   TBD
+Debian 9         | TBD   |   TBD
```
Comment (sandoche2k) [inline README.md]: Debian 9 mysql-server                  5.5.9999+default

File: README.md [hunk 3]
```diff
@@ -76,11 +74,13 @@ version 9.2 or earlier will _NOT_ work.
 Operating System | Perl
 ---------------- | ----
 CentOS 7         | 5.16                        
-Debian 7         | 5.14
-FreeBSD 10.3     | 5.24
-FreeBSD 11.1     | 5.24
+Debian 8         | TBD
+Debian 9         | TBD
```
Comment (sandoche2k) [inline README.md]: Perl => 5.24

**Comment by matsduf** [comment]
@mattias-p and @sandoche2k, I resolved the conflict and then your approvals are gone. Please review and approve again.

File: README.md (hunk 2 above)
Comment (matsduf) [reply README.md]: @sandoche2k, does Debian 9 come with MySQL version 5.5.9999? Do you know what version of PostgreSQL? And Debian 8?

File: README.md (hunk 3 above)
Comment (matsduf) [reply README.md]: @sandoche2k, do you know what version of Perl that comes with Debian 8 and Debian 9?

**Comment by mattias-p** [comment]
Shouldn't the TBD be filled in?

**Comment by matsduf** [comment]
Yes, I would not merge without having that data.

File: README.md (hunk 2 above)
Comment (sandoche2k) [reply README.md]: For Debian 8 
Mysql - 5.5.60
PostgreSQL- 9.4.15

For Debian 9
Mysql - mysql  Ver 15.1 Distrib 10.1.26-MariaDB (explanation here : https://mariadb.org/debian-9-released-mariadb-mysql-variant)
sh ```
root@vps323915:~# mysql -p
Enter password: 
Welcome to the MariaDB monitor.  Commands end with ; or \g.
Your MariaDB connection id is 4
Server version: 10.1.26-MariaDB-0+deb9u1 Debian 9.1
```
PostgreSQL- 9.6.7

File: README.md (hunk 3 above)
Comment (sandoche2k) [reply README.md]: Debian 8 => 5.20
Debian 9 => 5.24

File: README.md [hunk 4]
```diff
@@ -42,24 +42,26 @@ architecture listed below.
...
 
 ### Supported database engine versions
 
 Operating System | MySQL | PostgreSQL
 ---------------- | ------| -----------
 CentOS 7         | 5.6   |   9.3
-Debian 7         | 5.5   |  10.1
+Debian 8         | TBD   |   TBD
+Debian 9         | TBD   |   TBD
 FreeBSD 10.3     | 5.6   |   9.5
```
Comment (mattias-p) [inline README.md]: Should be updated to 10.4.

**Comment by matsduf** [comment]
@sandoche2k and @mattias-p, please review and approve, if possible.

**Pull request merged**